spec/integration/orders_spec.rb

```diff
@@ -26,16 +26,24 @@
   end
 
   it 'supports create with a project-id' do
-    retrieve_projects_response = Patch::Project.retrieve_projects(page: 1)
-    project_id = retrieve_projects_response.data.first.id
+    # biomass project
+    retrieve_project_response = Patch::Project.retrieve_project("pro_test_c3a9feba769fc7a8806377266ca9ff6a")
```

Comment:
nit: do we want to keep this id in a shared location so we can reuse it in other specs?